src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1604,11 +1584,16 @@ void TorrentImpl::reload()
         p.flags &= ~(lt::torrent_flags::auto_managed | lt::torrent_flags::paused);
     }
 
+    auto *extensionData = new ExtensionData;
```

Comment:
Previously this was `const`, why is it removed?

Reply:
> Previously this was `const`, why is it removed?

Well, changed to `const`.

Reply:
maybe something went wrong on you side? I didn't see the update...

Reply:
> maybe something went wrong on you side? I didn't see the update...

Damn. Forgot to push it.

Reply:
Shouldn't it be `const auto *`?

Reply:
It can't. Because then it can't be cast to non-const pointer when constructing `LTClientData` below.